# Service to get a list of stations from rosparam
---
# List of stations
Station[] station_list
---

